src/calculateRentalCost.js

```diff
@@ -1,10 +1,24 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
+  let totalAmount = days * dayCosts;
+  const longTermLoan = 7;
+  const longTermDiscount = 50;
+  const mediumTermLoan = 3;
+  const mediumTermDiscount = 20;
+
+  if (days >= longTermLoan) {
+    totalAmount -= longTermDiscount;
+
+    return totalAmount;
+  }
+
+  if (days >= mediumTermLoan) {
+    totalAmount -= mediumTermDiscount;
+
+    return totalAmount;
+  }
```

Comment:
The current logic applies the long-term discount and then immediately returns the total amount, preventing the medium-term discount from being considered. This is correct if the intention is to apply only one discount, but if both discounts should be considered when applicable, the logic needs adjustment. Ensure that the medium-term discount is not applied if the long-term discount is already applied.